pkgs/by-name/ch/chirpstack-mqtt-forwarder/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  darwin,
+  rustPlatform,
+  fetchFromGitHub,
+  gitUpdater,
+  versionCheckHook,
+  protobuf,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "chirpstack-mqtt-forwarder";
+  version = "4.3.1";
+
+  src = fetchFromGitHub {
+    owner = "chirpstack";
+    repo = "chirpstack-mqtt-forwarder";
+    rev = "v${version}";
+    hash = "sha256-jbu8O1Wag6KpN49VyXsYO8os95ctZjzuxKXoDMLyiKU=";
+  };
+
+  cargoHash = "sha256-1tAZjsjoVKUkrF0WAqxs9d+1w8/AqFGDfpFGAHvf+D0=";
+
+  nativeBuildInputs = [ protobuf ];
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+
+  # Skip tests depending on internet connectivity
+  checkFlags = [ "--skip=end_to_end" ];
+  doInstallCheck = true;
+
+  passthru.updateScript = gitUpdater { rev-prefix = "v"; };
```

Comment:
I don't think the gitUpdater updates cargoHash. Try
```
updateScript = nix-update-script { extraArgs = [ "--version=branch" ]; };
```

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/353917/commits/0e70d9b61b70791889b87d0833c4605e11a58a2a